lib/testScenario/blobUploader.ts

```diff
@@ -0,0 +1,37 @@
+import { BlobServiceClient } from "@azure/storage-blob";
+import { inject, injectable } from "inversify";
+import { TYPES } from "../inversifyUtils";
+import { setDefaultOpts } from "./../swagger/loader";
+export interface TestScenarioBlobUploaderOption {
+  blobConnectionString?: string;
+  enableBlobUploader?: boolean;
+}
+
+type ContainName = "postmancollection" | "newmanreport" | "report";
+
+@injectable()
+export class BlobUploader {
+  private blobServiceClient?: BlobServiceClient;
+  // eslint-disable-next-line @typescript-eslint/explicit-member-accessibility
+  constructor(@inject(TYPES.opts) private opts: TestScenarioBlobUploaderOption) {
+    setDefaultOpts(this.opts, {
+      enableBlobUploader: false,
+      blobConnectionString: "",
+    });
+    if (opts.enableBlobUploader) {
+      this.blobServiceClient = BlobServiceClient.fromConnectionString(
+        this.opts.blobConnectionString!
```

Comment:
How do we get the connectionString? we can use environment variable firstly then consider to use keyvault.

Reply:
connect to keyvalut will be a seprate step.